drivers/virtualbox/vm.go

```diff
@@ -0,0 +1,55 @@
+package virtualbox
+
+import (
+	"bufio"
+	"io"
+	"strconv"
+	"strings"
+)
+
+type VirtualBoxVM struct {
+	CPUs   int
+	Memory int
+}
+
+func parseVMInfo(r io.Reader) (*VirtualBoxVM, error) {
+	s := bufio.NewScanner(r)
+	vm := &VirtualBoxVM{}
+	for s.Scan() {
+		line := s.Text()
+		if line == "" {
+			continue
+		}
+		res := reEqualLine.FindStringSubmatch(line)
+		if res == nil {
+			continue
+		}
+		switch key, val := res[1], res[2]; key {
```

Comment:
Hm, I think I'd prefer separate lines for the assignment and the switch here for clarity (`switch key`) -- do you agree?